twake/backend/node/src/core/platform/services/push/connectors/fcm/service.ts

```diff
@@ -0,0 +1,47 @@
+import { option } from "yargs";
+import { logger } from "../../../../framework";
+import { PushConfiguration, PushMessageNotification, PushMessageOptions } from "../../types";
+import { PushConnector } from "../connector";
+
+export default class FcmPushConnector implements PushConnector {
+  name = "FcmPushConnector";
+
+  constructor(readonly configuration: PushConfiguration["fcm"]) {}
+
+  async push(
+    devices: string[],
```

Comment:
You should assert than devices is defined, just because at runtime it can, and it will break your loop.

Reply:
Done